.pipelines/templates/publish-helm-chart.yaml

```diff
@@ -1,15 +1,41 @@
+parameters: 
+  - name: releaseTag
+    type: boolean
+    default: false
+
 steps: 
   - bash: |
       az login --identity > /dev/null 2>&1
       az acr login --name $(REGISTRY_NAME) > /dev/null 2>&1
     displayName: Login to acr
     env: 
       REGISTRY_NAME: $(REGISTRY_NAME)
+  - script: |
+      cp $(image.dir)/$(chart.name)-*.tgz .
+      FILENAME=$(ls $(chart.name)-*.tgz)
+      TRIM=${FILENAME##$(chart.name)-}
+      echo "##vso[task.setvariable variable=CHART_TAG]${TRIM%%.tgz}"
+      echo ${CHART_TAG}
+    displayName: Get osm-arc chart version
```

Comment:
Can you make this more descriptive? Like "get osm-arc chart version from built and published chart"